src/commcare_cloud/commcare_cloud.py

```diff
@@ -148,9 +148,67 @@ def make_parser(available_envs, formatter_class=RawTextHelpFormatter,
     return parser, subparsers, commands
 
 
+def make_changelog_parser():
+    # Parse the contents of the changelog dir
+    changelog_contents = []
+    files_to_ignore = ['0000-changelog.md', 'index.md']
+    changelog_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'docs', 'changelog')
+    sorted_files = _sort_files(changelog_dir)
+    for change_file_name in sorted_files:
+        if change_file_name not in files_to_ignore:
+            try:
+                with open(os.path.join(changelog_dir, change_file_name)) as change_file:
+                    change_context = ''
```

Comment:
Moving this code (that processes the file) into a separate function would make this more readable.